src/can-deploy.spec.ts

```diff
@@ -175,7 +175,10 @@ describe('CanDeploy Spec', () => {
       };
       const ding = canDeployFactory(opts);
```

Comment:
I know it's not related to this PR, but... why `ding`?

Reply:
I don't know. That's how it was when I got there.